src/client/hooks/UseFetchApi.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const UseFetchApi = (url) => {
```

Comment:
Usually hooks are camelCased:

```suggestion
export const useFetchApi = (url) => {
```

Reply:
Could you write a brief section in the Readme file with an example how to use this.